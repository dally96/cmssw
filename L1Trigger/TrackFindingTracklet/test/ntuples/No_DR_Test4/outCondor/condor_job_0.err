************************************************************************************************************************
						 WARNING!
 The key phase2_realistic now points to 113X_mcRun4_realistic_v7.
 Usage of this key is discretionary and users are cautioned to use it at their own risk!
 This Global Tag contains tags suited for an Inner Tracker layout with rectangular (25x100um2) pixel cells (T15,T21).
 If this is not the geometry you are expecting to use, please consult:
 https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoCondPhase2.py to retrieve the right key.
************************************************************************************************************************
17-Feb-2022 16:41:59 CET  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/81b36c94-be96-43b3-9868-b98a9410899d.root
%MSG-w XrdAdaptorInternal:  file_open 17-Feb-2022 16:42:00 CET pre-events
Failed to open file at URL root://eoscms.cern.ch:1094//eos/cms/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/81b36c94-be96-43b3-9868-b98a9410899d.root?xrdcl.requuid=683dc602-d613-43d6-b8a0-8cd55cf9b3e2.
%MSG
%MSG-w XrdAdaptorInternal:  file_open 17-Feb-2022 16:42:00 CET pre-events
Failed to open file at URL root://eoscms.cern.ch:1094//eos/cms/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/81b36c94-be96-43b3-9868-b98a9410899d.root?tried=&xrdcl.requuid=28da56c1-c20b-4eac-969e-702f3664c0b2.
%MSG
17-Feb-2022 16:42:00 CET  Initiating request to open file root://xrootd-cms.infn.it//store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/81b36c94-be96-43b3-9868-b98a9410899d.root
%MSG-w XrdAdaptor:  file_open 17-Feb-2022 16:42:02 CET pre-events
Data is served from T2_UK_SGrid_RALPP instead of original site T2_FR_GRIF_LLR
%MSG
17-Feb-2022 16:42:06 CET  Successfully opened file root://xrootd-cms.infn.it//store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/81b36c94-be96-43b3-9868-b98a9410899d.root
L1TrackNtupleMaker::beginJob
%MSG-w XrdAdaptor:  (NoModuleName) 17-Feb-2022 16:46:10 CET pre-events
Data is now served from T2_UK_SGrid_RALPP, [2001 instead of previous T2_UK_SGrid_RALPP
%MSG
%MSG-w XrdAdaptor:  AfterModBeginStream 17-Feb-2022 16:50:48 CET BeforeEvents
Data is now served from [2001 instead of previous T2_UK_SGrid_RALPP, [2001
%MSG
[2022-02-17 16:51:25.015659 +0100][Error  ][PostMaster        ] [t2-xrdcms.lnl.infn.it:7070 #0] Forcing error on disconnect: [ERROR] Operation interrupted.
[2022-02-17 16:51:25.015977 +0100][Error  ][PostMaster        ] [t2-xrdcms.lnl.infn.it:1094 #0] Forcing error on disconnect: [ERROR] Operation interrupted.
Begin processing the 1st record. Run 1, Event 408, LumiSection 5 on stream 0 at 17-Feb-2022 16:55:59.159 CET
Begin processing the 2nd record. Run 1, Event 401, LumiSection 5 on stream 0 at 17-Feb-2022 16:56:07.174 CET
Begin processing the 3rd record. Run 1, Event 404, LumiSection 5 on stream 0 at 17-Feb-2022 16:56:15.246 CET
Begin processing the 4th record. Run 1, Event 405, LumiSection 5 on stream 0 at 17-Feb-2022 16:56:21.992 CET
Begin processing the 5th record. Run 1, Event 402, LumiSection 5 on stream 0 at 17-Feb-2022 16:56:28.854 CET
Begin processing the 6th record. Run 1, Event 406, LumiSection 5 on stream 0 at 17-Feb-2022 16:56:36.452 CET
Begin processing the 7th record. Run 1, Event 410, LumiSection 5 on stream 0 at 17-Feb-2022 16:56:45.806 CET
Begin processing the 8th record. Run 1, Event 403, LumiSection 5 on stream 0 at 17-Feb-2022 16:56:54.256 CET
Begin processing the 9th record. Run 1, Event 411, LumiSection 5 on stream 0 at 17-Feb-2022 16:57:01.445 CET
Begin processing the 10th record. Run 1, Event 409, LumiSection 5 on stream 0 at 17-Feb-2022 16:57:08.632 CET
Begin processing the 11th record. Run 1, Event 413, LumiSection 5 on stream 0 at 17-Feb-2022 16:57:15.942 CET
Begin processing the 12th record. Run 1, Event 407, LumiSection 5 on stream 0 at 17-Feb-2022 16:57:23.895 CET
Begin processing the 13th record. Run 1, Event 418, LumiSection 5 on stream 0 at 17-Feb-2022 16:57:31.898 CET
Begin processing the 14th record. Run 1, Event 415, LumiSection 5 on stream 0 at 17-Feb-2022 16:57:38.311 CET
Begin processing the 15th record. Run 1, Event 417, LumiSection 5 on stream 0 at 17-Feb-2022 16:57:44.625 CET
Begin processing the 16th record. Run 1, Event 414, LumiSection 5 on stream 0 at 17-Feb-2022 16:57:51.512 CET
Begin processing the 17th record. Run 1, Event 416, LumiSection 5 on stream 0 at 17-Feb-2022 16:58:00.364 CET
Begin processing the 18th record. Run 1, Event 412, LumiSection 5 on stream 0 at 17-Feb-2022 16:58:07.976 CET
Begin processing the 19th record. Run 1, Event 423, LumiSection 5 on stream 0 at 17-Feb-2022 16:58:15.282 CET
Begin processing the 20th record. Run 1, Event 420, LumiSection 5 on stream 0 at 17-Feb-2022 16:58:23.722 CET
Begin processing the 21st record. Run 1, Event 419, LumiSection 5 on stream 0 at 17-Feb-2022 16:58:29.912 CET
Begin processing the 22nd record. Run 1, Event 421, LumiSection 5 on stream 0 at 17-Feb-2022 16:58:37.214 CET
Begin processing the 23rd record. Run 1, Event 426, LumiSection 5 on stream 0 at 17-Feb-2022 16:58:45.330 CET
Begin processing the 24th record. Run 1, Event 424, LumiSection 5 on stream 0 at 17-Feb-2022 16:58:53.613 CET
Begin processing the 25th record. Run 1, Event 422, LumiSection 5 on stream 0 at 17-Feb-2022 16:59:01.599 CET
Begin processing the 26th record. Run 1, Event 430, LumiSection 5 on stream 0 at 17-Feb-2022 16:59:10.183 CET
Begin processing the 27th record. Run 1, Event 429, LumiSection 5 on stream 0 at 17-Feb-2022 16:59:17.273 CET
Begin processing the 28th record. Run 1, Event 425, LumiSection 5 on stream 0 at 17-Feb-2022 16:59:25.498 CET
Begin processing the 29th record. Run 1, Event 432, LumiSection 5 on stream 0 at 17-Feb-2022 16:59:33.156 CET
Begin processing the 30th record. Run 1, Event 427, LumiSection 5 on stream 0 at 17-Feb-2022 16:59:41.575 CET
Begin processing the 31st record. Run 1, Event 436, LumiSection 5 on stream 0 at 17-Feb-2022 16:59:51.740 CET
Begin processing the 32nd record. Run 1, Event 431, LumiSection 5 on stream 0 at 17-Feb-2022 16:59:58.380 CET
Begin processing the 33rd record. Run 1, Event 435, LumiSection 5 on stream 0 at 17-Feb-2022 17:00:07.908 CET
Begin processing the 34th record. Run 1, Event 437, LumiSection 5 on stream 0 at 17-Feb-2022 17:00:17.440 CET
Begin processing the 35th record. Run 1, Event 433, LumiSection 5 on stream 0 at 17-Feb-2022 17:00:26.223 CET
Begin processing the 36th record. Run 1, Event 434, LumiSection 5 on stream 0 at 17-Feb-2022 17:00:34.372 CET
Begin processing the 37th record. Run 1, Event 438, LumiSection 5 on stream 0 at 17-Feb-2022 17:00:42.964 CET
Begin processing the 38th record. Run 1, Event 440, LumiSection 5 on stream 0 at 17-Feb-2022 17:00:51.056 CET
Begin processing the 39th record. Run 1, Event 439, LumiSection 5 on stream 0 at 17-Feb-2022 17:00:58.690 CET
Begin processing the 40th record. Run 1, Event 441, LumiSection 5 on stream 0 at 17-Feb-2022 17:01:08.916 CET
Begin processing the 41st record. Run 1, Event 443, LumiSection 5 on stream 0 at 17-Feb-2022 17:01:19.450 CET
Begin processing the 42nd record. Run 1, Event 446, LumiSection 5 on stream 0 at 17-Feb-2022 17:01:27.108 CET
Begin processing the 43rd record. Run 1, Event 447, LumiSection 5 on stream 0 at 17-Feb-2022 17:01:32.298 CET
Begin processing the 44th record. Run 1, Event 445, LumiSection 5 on stream 0 at 17-Feb-2022 17:01:39.516 CET
Begin processing the 45th record. Run 1, Event 444, LumiSection 5 on stream 0 at 17-Feb-2022 17:01:46.641 CET
Begin processing the 46th record. Run 1, Event 449, LumiSection 5 on stream 0 at 17-Feb-2022 17:01:54.250 CET
Begin processing the 47th record. Run 1, Event 450, LumiSection 5 on stream 0 at 17-Feb-2022 17:02:00.617 CET
Begin processing the 48th record. Run 1, Event 428, LumiSection 5 on stream 0 at 17-Feb-2022 17:02:07.988 CET
[2022-02-17 17:02:15.438628 +0100][Error  ][PostMaster        ] [eoscms.cern.ch:1094 #0] Forcing error on disconnect: [ERROR] Operation interrupted.
Begin processing the 49th record. Run 1, Event 442, LumiSection 5 on stream 0 at 17-Feb-2022 17:02:16.368 CET
Begin processing the 50th record. Run 1, Event 453, LumiSection 5 on stream 0 at 17-Feb-2022 17:02:26.053 CET
Begin processing the 51st record. Run 1, Event 448, LumiSection 5 on stream 0 at 17-Feb-2022 17:02:34.180 CET
Begin processing the 52nd record. Run 1, Event 452, LumiSection 5 on stream 0 at 17-Feb-2022 17:02:44.080 CET
Begin processing the 53rd record. Run 1, Event 454, LumiSection 5 on stream 0 at 17-Feb-2022 17:02:52.440 CET
Begin processing the 54th record. Run 1, Event 456, LumiSection 5 on stream 0 at 17-Feb-2022 17:03:01.866 CET
[2022-02-17 17:03:03.011855 +0100][Error  ][AsyncSock         ] [xrootd.ba.infn.it:1094 #0.0] Socket error while handshaking: [ERROR] Socket timeout
[2022-02-17 17:03:03.012033 +0100][Error  ][PostMaster        ] [xrootd.ba.infn.it:1094 #0] elapsed = 45, pConnectionWindow = 31 seconds.
Begin processing the 55th record. Run 1, Event 455, LumiSection 5 on stream 0 at 17-Feb-2022 17:03:10.026 CET
Begin processing the 56th record. Run 1, Event 451, LumiSection 5 on stream 0 at 17-Feb-2022 17:03:19.387 CET
Begin processing the 57th record. Run 1, Event 458, LumiSection 5 on stream 0 at 17-Feb-2022 17:03:27.620 CET
Begin processing the 58th record. Run 1, Event 463, LumiSection 5 on stream 0 at 17-Feb-2022 17:03:37.329 CET
Begin processing the 59th record. Run 1, Event 459, LumiSection 5 on stream 0 at 17-Feb-2022 17:03:44.800 CET
[2022-02-17 17:03:48.002811 +0100][Error  ][AsyncSock         ] [xrootd.ba.infn.it:1094 #0.0] Socket error while handshaking: [ERROR] Socket timeout
[2022-02-17 17:03:48.002880 +0100][Error  ][PostMaster        ] [xrootd.ba.infn.it:1094 #0] elapsed = 45, pConnectionWindow = 31 seconds.
[2022-02-17 17:03:48.002956 +0100][Error  ][PostMaster        ] [xrootd.ba.infn.it:1094 #0] Unable to recover: [ERROR] Socket timeout.
[2022-02-17 17:03:48.002965 +0100][Error  ][XRootD            ] [xrootd.ba.infn.it:1094] Impossible to send message kXR_open (file: /store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/81b36c94-be96-43b3-9868-b98a9410899d.root?tried=+1213xrootd-cms-redir-int.cr.cnaf.infn.it,+1313xrootd-redic.pi.infn.it,+1213xrootd011213xrootd-cms-uk.gridpp.rl1213xrootd02.grid.hep.ph.ic.ac.uk,+1213llrxrd-redir.in2p3.fr,+1313llrxrd-redir.in2p3.fr,t2-xrdcms.lnl.infn.it,heplnx228.pp.rl.ac.uk,heplnx229.pp.rl.ac.uk,xrootd-cms.infn.it,xrootd-redic.pi.infn.it&triedrc=enoent, mode: 0460, flags: kXR_open_read kXR_async kXR_retstat ). Trying to recover.
Begin processing the 60th record. Run 1, Event 460, LumiSection 5 on stream 0 at 17-Feb-2022 17:03:54.932 CET
Begin processing the 61st record. Run 1, Event 461, LumiSection 5 on stream 0 at 17-Feb-2022 17:04:03.804 CET
Begin processing the 62nd record. Run 1, Event 457, LumiSection 5 on stream 0 at 17-Feb-2022 17:04:11.913 CET
Begin processing the 63rd record. Run 1, Event 465, LumiSection 5 on stream 0 at 17-Feb-2022 17:04:20.164 CET
Begin processing the 64th record. Run 1, Event 462, LumiSection 5 on stream 0 at 17-Feb-2022 17:04:27.642 CET
Begin processing the 65th record. Run 1, Event 464, LumiSection 5 on stream 0 at 17-Feb-2022 17:04:35.859 CET
Begin processing the 66th record. Run 1, Event 467, LumiSection 5 on stream 0 at 17-Feb-2022 17:04:42.426 CET
Begin processing the 67th record. Run 1, Event 468, LumiSection 5 on stream 0 at 17-Feb-2022 17:04:50.138 CET
Begin processing the 68th record. Run 1, Event 466, LumiSection 5 on stream 0 at 17-Feb-2022 17:04:58.866 CET
Begin processing the 69th record. Run 1, Event 471, LumiSection 5 on stream 0 at 17-Feb-2022 17:05:08.044 CET
Begin processing the 70th record. Run 1, Event 472, LumiSection 5 on stream 0 at 17-Feb-2022 17:05:13.710 CET
Begin processing the 71st record. Run 1, Event 470, LumiSection 5 on stream 0 at 17-Feb-2022 17:05:21.519 CET
Begin processing the 72nd record. Run 1, Event 475, LumiSection 5 on stream 0 at 17-Feb-2022 17:05:31.437 CET
Begin processing the 73rd record. Run 1, Event 469, LumiSection 5 on stream 0 at 17-Feb-2022 17:05:38.217 CET
Begin processing the 74th record. Run 1, Event 474, LumiSection 5 on stream 0 at 17-Feb-2022 17:05:46.545 CET
Begin processing the 75th record. Run 1, Event 477, LumiSection 5 on stream 0 at 17-Feb-2022 17:05:54.195 CET
Begin processing the 76th record. Run 1, Event 478, LumiSection 5 on stream 0 at 17-Feb-2022 17:06:01.916 CET
Begin processing the 77th record. Run 1, Event 473, LumiSection 5 on stream 0 at 17-Feb-2022 17:06:11.011 CET
Begin processing the 78th record. Run 1, Event 476, LumiSection 5 on stream 0 at 17-Feb-2022 17:06:18.593 CET
Begin processing the 79th record. Run 1, Event 480, LumiSection 5 on stream 0 at 17-Feb-2022 17:06:25.027 CET
Begin processing the 80th record. Run 1, Event 481, LumiSection 5 on stream 0 at 17-Feb-2022 17:06:32.802 CET
Begin processing the 81st record. Run 1, Event 484, LumiSection 5 on stream 0 at 17-Feb-2022 17:06:42.049 CET
Begin processing the 82nd record. Run 1, Event 485, LumiSection 5 on stream 0 at 17-Feb-2022 17:06:50.150 CET
Begin processing the 83rd record. Run 1, Event 483, LumiSection 5 on stream 0 at 17-Feb-2022 17:06:58.772 CET
Begin processing the 84th record. Run 1, Event 479, LumiSection 5 on stream 0 at 17-Feb-2022 17:07:06.896 CET
Begin processing the 85th record. Run 1, Event 491, LumiSection 5 on stream 0 at 17-Feb-2022 17:07:16.511 CET
Begin processing the 86th record. Run 1, Event 490, LumiSection 5 on stream 0 at 17-Feb-2022 17:07:22.687 CET
Begin processing the 87th record. Run 1, Event 482, LumiSection 5 on stream 0 at 17-Feb-2022 17:07:30.901 CET
Begin processing the 88th record. Run 1, Event 486, LumiSection 5 on stream 0 at 17-Feb-2022 17:07:39.761 CET
Begin processing the 89th record. Run 1, Event 492, LumiSection 5 on stream 0 at 17-Feb-2022 17:07:48.013 CET
Begin processing the 90th record. Run 1, Event 493, LumiSection 5 on stream 0 at 17-Feb-2022 17:07:56.380 CET
Begin processing the 91st record. Run 1, Event 494, LumiSection 5 on stream 0 at 17-Feb-2022 17:08:06.490 CET
Begin processing the 92nd record. Run 1, Event 489, LumiSection 5 on stream 0 at 17-Feb-2022 17:08:12.742 CET
Begin processing the 93rd record. Run 1, Event 495, LumiSection 5 on stream 0 at 17-Feb-2022 17:08:21.519 CET
Begin processing the 94th record. Run 1, Event 488, LumiSection 5 on stream 0 at 17-Feb-2022 17:08:29.022 CET
Begin processing the 95th record. Run 1, Event 499, LumiSection 5 on stream 0 at 17-Feb-2022 17:08:38.928 CET
Begin processing the 96th record. Run 1, Event 496, LumiSection 5 on stream 0 at 17-Feb-2022 17:08:48.033 CET
Begin processing the 97th record. Run 1, Event 498, LumiSection 5 on stream 0 at 17-Feb-2022 17:08:53.875 CET
Begin processing the 98th record. Run 1, Event 487, LumiSection 5 on stream 0 at 17-Feb-2022 17:09:03.075 CET
Begin processing the 99th record. Run 1, Event 497, LumiSection 5 on stream 0 at 17-Feb-2022 17:09:12.866 CET
Begin processing the 100th record. Run 1, Event 500, LumiSection 5 on stream 0 at 17-Feb-2022 17:09:21.901 CET
17-Feb-2022 17:09:28 CET  Closed file root://xrootd-cms.infn.it//store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/81b36c94-be96-43b3-9868-b98a9410899d.root
L1TrackNtupleMaker::endJob
TimeReport> Time report complete in 1667.95 seconds
 Time Summary: 
 - Min event:   4.82579
 - Max event:   9.44199
 - Avg event:   6.96743
 - Total loop:  1641.24
 - Total init:  14.4756
 - Total job:   1667.95
 - EventSetup Lock:   0
 - EventSetup Get:   0
 Event Throughput: 0.0609294 ev/s
 CPU Summary: 
 - Total loop:  566.248
 - Total init:  7.30369
 - Total extra: 0
 - Total job:   581.73
 Processing Summary: 
 - Number of Events:  100
 - Number of Global Begin Lumi Calls:  1
 - Number of Global Begin Run Calls: 1


=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 XrdAdaptor           -w (NoModuleName)                         1        1
    2 XrdAdaptor           -w AfterModBeginStr                       1        1
    3 XrdAdaptor           -w file_open                              1        1
    4 XrdAdaptorInternal   -w file_open                              2        2
    5 TimeReport           -e AfterModEndJob                         1        1
    6 fileAction           -s file_close                             1        1
    7 fileAction           -s file_open                              3        3

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 XrdAdaptor           pre-events                        
    2 XrdAdaptor           BeforeEvents                      
    3 XrdAdaptor           pre-events                        
    4 XrdAdaptorInternal   pre-events       pre-events       
    5 TimeReport           EndJob                            
    6 fileAction           PostGlobalEndRun                  
    7 fileAction           pre-events       pre-events       pre-events

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 5                   5
Error                   1                   1
System                  4                   4

dropped waiting message count 0
