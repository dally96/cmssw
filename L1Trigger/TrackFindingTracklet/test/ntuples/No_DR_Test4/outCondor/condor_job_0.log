000 (803401.000.000) 02/17 16:39:43 Job submitted from host: <188.185.121.235:9618?addrs=188.185.121.235-9618+[2001-1458-d00-49--100-36e]-9618&alias=bigbird22.cern.ch&noUDP&sock=schedd_8027_6ecd>
...
040 (803401.000.000) 02/17 16:41:33 Entered queue to transfer input files
	Transferring to host: <188.184.163.12:9618?addrs=188.184.163.12-9618+[2001-1458-301-4d--100-6]-9618&alias=b7s12p0906.cern.ch&noUDP&sock=slot1_8_3892_0417_63110>
...
040 (803401.000.000) 02/17 16:41:49 Started transferring input files
	Seconds spent in queue: 16
...
040 (803401.000.000) 02/17 16:41:49 Finished transferring input files
...
001 (803401.000.000) 02/17 16:41:50 Job executing on host: <188.184.163.12:9618?addrs=188.184.163.12-9618+[2001-1458-301-4d--100-6]-9618&alias=b7s12p0906.cern.ch&noUDP&sock=startd_3073_669f>
...
006 (803401.000.000) 02/17 16:41:59 Image size of job updated: 840740
	159  -  MemoryUsage of job (MB)
	161804  -  ResidentSetSize of job (KB)
...
006 (803401.000.000) 02/17 16:46:59 Image size of job updated: 2058432
	393  -  MemoryUsage of job (MB)
	401516  -  ResidentSetSize of job (KB)
...
006 (803401.000.000) 02/17 16:57:00 Image size of job updated: 3073420
	845  -  MemoryUsage of job (MB)
	864780  -  ResidentSetSize of job (KB)
...
006 (803401.000.000) 02/17 17:02:01 Image size of job updated: 3073484
	845  -  MemoryUsage of job (MB)
	864780  -  ResidentSetSize of job (KB)
...
006 (803401.000.000) 02/17 17:07:01 Image size of job updated: 3073484
	860  -  MemoryUsage of job (MB)
	879920  -  ResidentSetSize of job (KB)
...
006 (803401.000.000) 02/17 17:09:41 Image size of job updated: 3237324
	864  -  MemoryUsage of job (MB)
	884636  -  ResidentSetSize of job (KB)
...
040 (803401.000.000) 02/17 17:09:41 Started transferring output files
	Seconds spent in queue: 1688
...
040 (803401.000.000) 02/17 17:09:41 Finished transferring output files
...
005 (803401.000.000) 02/17 17:09:41 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:07:21, Sys 0 00:02:20  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:07:21, Sys 0 00:02:20  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	25498  -  Run Bytes Sent By Job
	25494  -  Run Bytes Received By Job
	25498  -  Total Bytes Sent By Job
	25494  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.51        1         1 
	   Disk (KB)            :    71          17   3043479 
	   Memory (MB)          :   864        2000      2000 

	Job terminated of its own accord at 2022-02-17T16:09:41Z.
...
