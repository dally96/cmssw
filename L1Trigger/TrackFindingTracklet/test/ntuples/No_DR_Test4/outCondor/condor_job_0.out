/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test /pool/condor/dir_130750 
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=1828645717
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 10:42:20
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 10:42:20
uri       : voms-cms-auth.app.cern.ch:443

subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=1828645717
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 10:42:20
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 10:42:20
uri       : voms-cms-auth.app.cern.ch:443

cmsRun /afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/customise_L1TrackNtupleMaker.py inputFiles_clear inputFiles_load=/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/ntuples/No_DR_Test4/pFiles_0.txt outputFile=outputL1Ntuple_0.root print
Singletons:
  filePrepend        : 
                     - String to prepend location of all files
  maxEvents          : -1
                     - Number of events to process (-1 for all)
  outputFile         : outputL1Ntuple_0.root
                     - Name of output file (if needed)
  secondaryOutputFile: 
                     - Name of second output file (if needed)
  section            : 0
                     - This section (from 1..totalSections inclusive)
  tag                : 
                     - tag to add to output filename
  totalSections      : 0
                     - Total number of sections
Lists:
  inputFiles         : ['/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/81b36c94-be96-43b3-9868-b98a9410899d.root' ]
                     - Files to process
  secondaryInputFiles: []
                     - Second group of files to process (if needed)

using geometry D76 (tilted)
